_data/projects/search/releases/7.0/series.yml

```
@@ -0,0 +1,38 @@
+summary: >-
+  JDK 11 as a baseline,
+  switch to Jakarta EE, compatibility with ORM 6.3 and no `-orm6` artifacts, 
+  upgrade to Lucene 9 in the Hibernate Search Lucene backend,
+  published Hibernate Search BOM,
+  experimental compatibility with Amazon OpenSearch Serverless,
+  drop compatibility with EOLed Elasticsearch versions 5.x - 6.x
+maven:
+  artifacts:
+    - artifact_id: hibernate-search-mapper-orm
+      summary: Hibernate ORM mapper
+    - artifact_id: hibernate-search-mapper-orm-coordination-outbox-polling
+      summary: "\"outbox-polling\" coordination strategy for the Hibernate ORM mapper"
+    - artifact_id: hibernate-search-mapper-pojo-standalone
+      summary: Standalone POJO mapper
+    - artifact_id: hibernate-search-backend-lucene
+      summary: Lucene backend
+    - artifact_id: hibernate-search-backend-elasticsearch
+      summary: Elasticsearch/OpenSearch backend
+    - artifact_id: hibernate-search-backend-elasticsearch-aws
+      summary: Amazon IAM authentication for Elasticsearch/OpenSearch
+    - artifact_id: hibernate-search-mapper-orm-batch-jsr352-core
+      summary: Batch for Java (JSR-352) mass indexing job for the Hibernate ORM mapper - Core
+    - artifact_id: hibernate-search-mapper-orm-batch-jsr352-jberet
+      summary: Batch for Java (JSR-352) mass indexing job for the Hibernate ORM mapper - JBeret specifics
+    - artifact_id: hibernate-search-v5migrationhelper-orm
+      summary: Helper for migrating from Hibernate Search 5 to Hibernate Search 6 (Hibernate ORM mapper + Lucene backend)
+integration_constraints:
+  java:
+    version: 11, 17, 20, 21 or 22
```

Let's not mention 22 as it's not GA yet.
And I'd remove 20 since 21 is GA now.

```suggestion
    version: 11, 17 or 21
```